lambda/custom-collection/README.md

```diff
@@ -0,0 +1,86 @@
+# MetaKeep Lambda - Custom ERC721
+
+This repository contains a Custom ERC721 Application that you can use as the starting point
+for using MetaKeep's Lambda Infrastructure.
+
+This project is intended to be used with the
+[MetaKeep Lambda Creation and Invocation Tutorial Series](https://docs.metakeep.xyz/docs/create-your-first-lambda), but you should be
+able to follow it by yourself by reading the README and exploring its
+`contracts` and `scripts` directories.
+
+## Directory Structure
+
+The project is organized as follows:
+
+- [contracts](./contracts): Contains the Smart Contract for Custom ERC721 Tokens.
```

Comment:
this directory doesn't exist

Reply:
fixed ideally, it should be ./smart-contracts/contracts

Reply:
the folder name is still wrong, There is no `contracts` folder